frontend/src/pages/bceidform/BusinessInformationWizardStep.vue

```diff
@@ -269,6 +269,11 @@ watch([detailsData], () => {
     const forestClientDetails: ForestClientDetailsDto = detailsData.value;
     
     if (!features.BCEID_MULTI_ADDRESS) {
+      // To prevent multiple addresses from breaking the validations
```

Comment:
How many address could we get here?
Is it guaranteed to be either 0, 1 or 2? I'm asking because, if it's more than 2 than this logic might not be enough.

Reply:
You're correct, it will be either 0, 1 or 2. Being 2 at most.